lib/jobs/generate-enclosure.js

```diff
@@ -203,5 +218,32 @@ function generateEnclosureJobFactory(
         });
     }
 
+    /**
+     * Get target nodes of the encloses or enclosedBy relation
```

Comment:
could you improve this to reflect what the method is doing? It's not just getting the node, but removing the relation as well. "pop" to me isn't clear as to the intent